samples/Sandbox.AspNetCore2_1_To_2_2/Sandbox.AspNetCore2_1_To_2_2.csproj

```diff
@@ -1,15 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
+    <PropertyGroup>
+      <TargetFramework>netcoreapp2.1</TargetFramework>
+      <RootNamespace>Sandbox</RootNamespace>
+    </PropertyGroup>
 
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-  </PropertyGroup>
+    <ItemGroup>
+      <PackageReference Include="Microsoft.AspNetCore.App" />
+    </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\Sandbox.Shared\Sandbox.Shared.csproj" />
-  </ItemGroup>
+    <ItemGroup>
+      <ProjectReference Include="..\Sandbox.Shared\Sandbox.Shared.csproj" />
+    </ItemGroup>
 
+    <ItemGroup>
```

Comment:
The sandbox projects all need the `<CodeAnalysisRuleSet>` and the analyzer package references; or should get stylecop.json removed. It doesn't do anything without the analyzers.

I think it'd probably be good to have the sandbox code consistent with the other product/test code, though. Probably.